core/src/main/java/tc/oc/pgm/goals/GoalMatchModule.java

```diff
@@ -147,7 +147,9 @@ protected void updateProgress(Goal goal) {
     for (Competitor competitor : competitorsByGoal.get(goal)) {
       progressByCompetitor.put(competitor, new GoalProgress(competitor));
     }
-    match.calculateVictory();
+
+    // Forces team rankings to be invalidated
+    match.getWinners();
```

Comment:
This is subtle behavior which is not great, it’s better if we could do something explicit. But I’ll still merge for now.